src/App.tsx

```diff
@@ -32,7 +32,12 @@ const App = () => {
               <Route path="/" element={<Home />} />
               <Route path="/tasks" element={<ActiveTasks />} />
               <Route path="/calendar" element={<Calendar />} />
-              <Route path="/leaderboard" element={<Leaderboard users={tribeUsers} currentUserID={userUID} />} />
+              <Route
+                path="/leaderboard"
+                element={
+                  <Leaderboard users={tribeUsers} currentUserID={userUID} />
+                }
+              />
```

Comment:
Does this need to be changed? I'm concerned about future merge conflicts with App.tsx

Reply:
Hi Todd, 

The majority of us have prettier set up on our machines to do format like this so I think we should keep it with this format.